docs/data_mover_backup_pvc_configuration.md

```diff
@@ -0,0 +1,70 @@
+# Advanced Configuration for the Data Movement
+
+This guide provides instructions for configuring intermediate PVC behavior used during backup and restore workflows in OADP via the Data Protection Application (DPA) Custom Resource Definition (CRD).
+
+The settings introduced here provide control over how NodeAgent handles intermediate persistent volumes used in data movement operations.
+
+The following sections describe how to configure:
+
+- `backupPVC`
+- `restorePVC`
+
+These settings are defined under the `.spec.configuration.nodeAgent` section of the DPA CRD.
+
+---
+
+## 1. Configuring backupPVC for Snapshot Data Movement
+
+The `backupPVC` field allows you to define behavior for intermediate PVCs used during snapshot-to-object-store data movement. You can configure multiple storage classes, each with specific parameters.
+
+Each storage class block within backupPVC supports the following fields:
+
+- `storageClass` – the name of the storage class to use for the intermediate PVC.
+- `readOnly` – indicates if the PVC should be mounted as read-only. In some storage providers this can significantly reduce the performance when creating the backup from the snapshot. Setting this option to true requires `spcNoRelabeling` to be set to true as well.
+- `spcNoRelabeling` – disables automatic relabeling of the SecurityContextConstraints if set to true. It must be set to true when `readOnly` is set to true.
+
+Please refer to the Velero documentation for more details on the `backupPVC` feature. In the OADP project, this feature is controlled via the `nodeAgent.backupPVC` field in the DPA CRD, not through a ConfigMap. The latter is automatically created and managed by the OADP controller for you: [BackupPVC Configuration for Data Movement Backup](https://velero.io/docs/main/data-movement-backup-pvc-configuration/).
+
+
+Example Configuration
+
+The following example configures two storage classes: `gp3-csi` and `another-storage-class`. The gp3-csi PVC is mounted read-write, while `another-storage-class` is mounted read-only.
```

Comment:
I think this is a bit confusing. Maybe use an example like:
```yaml
spec:
  configuration:
    nodeAgent:
      enable: true
      backupPVC:
        gp2-csi: <----- the source/app pvc sc
          readOnly: true
          spcNoRelabeling: true # required when readOnly is true
          storageClass: gp2-csi-readonly  <---- the sc to be used by backupPVC instead of the source/app pvc sc
        gp3-csi:
          readOnly: false
          spcNoRelabeling: false
          storageClass: mock-gp3-csi-something
```

Reply:
yeah works for me